build-utils/findmodifiedprojects.sh

```diff
@@ -3,8 +3,8 @@
 # Can run anywhere within the repo
 # If a file from a common area, like the root diretory was modified, we call that _SHARED_
 set +x
-#ignore these file name patterns when figuring out what modules changed 
-declare -a exclude_patterns=( '^.*\.md' '^.*.pdf' )
+#ignore these file name patterns when figuring out what modules changed
+declare -a exclude_patterns=( '^.*\.md' '^.*.pdf', '^.*\.spec\.ts' )
```

Comment:
Also exclude modified test files. Playwright E2E tests and Angular unit tests are included in new regex expression. 
It doesn't make any sense to deploy all studies after PR merge if only tests were changed in PR.